src/components/Menus/Menu.js

```diff
@@ -0,0 +1,204 @@
+import React             from 'react'
+import PropTypes         from 'prop-types'
+
+const styles = {
+  borderRadius: '4px',
+  backgroundColor: '#ffffff',
+  boxShadow: '0 3px 10px 0 rgba(0, 0, 0, 0.16), 0 3px 10px 0 rgba(0, 0, 0, 0.23)',
+  display: 'inline-block',
+  padding: '8px 0',
+  overflowY: 'auto',
+  userSelect: 'none',
+  outline: 'none'
+}
+
+class Menu extends React.Component {
+  static propTypes = {
+    /** aria-hidden HTML attribute */
+    ariaHidden: PropTypes.bool,
+    /** MenuItems or Divider */
+    children: PropTypes.node.isRequired,
+    /** The maximum height of the Menu when it is displayed */
+    maxHeight: PropTypes.number,
```

Comment:
I'm guessing this can't be controlled via the style object. Just curious what stops that from working?